agi_av_gb_abgleich_pub/build.gradle

```diff
@@ -3,14 +3,35 @@ import ch.so.agi.gretl.api.TransferSet
 
 apply plugin: 'ch.so.agi.gretl'
 
-defaultTasks 'transferAgiAvGbAbgleich'
+defaultTasks 'RemoveImportedData'
 
-task transferAgiAvGbAbgleich(type: Db2Db){
-    sourceDb = [dbUriSogis, dbUserSogis, dbPwdSogis]
+
+task transferGbData(type: Db2Db){
+    sourceDb = [dbUriGb, dbUserGb, dbPwdGb]
+    targetDb = [dbUriErf, dbUserErf, dbPwdErf]
+    transferSets = [
+            new TransferSet('agi_av_gb_abgleich_gb_daten.sql', 'agi_av_gb_abgleich.gb_daten', true)
+    ]
+}
+
+task calculateDifferences(type: Db2Db, dependsOn: 'transferGbData'){
+    sourceDb = [dbUriErf, dbUserErf, dbPwdErf]
     targetDb = [dbUriPub, dbUserPub, dbPwdPub]
     transferSets = [
-            new TransferSet('agi_av_gb_abgleich_pub_differenzen.sql', 'agi_av_gb_abgleich_pub.differenzen', true),
-            new TransferSet('agi_av_gb_abgleich_pub_uebersicht_des_vergleichs.sql',
-                    'agi_av_gb_abgleich_pub.uebersicht_des_vergleichs', true)
+            new TransferSet('agi_av_gb_abgleich_pub_differenzen.sql', 'agi_av_gb_abgleich_pub.differenzen', true)
     ];
 }
+
+task createOverview(type: SqlExecutor, dependsOn: 'calculateDifferences'){
+    database = [dbUriPub, dbUserPub, dbPwdPub]
+    sqlFiles = [
+            'agi_av_gb_abgleich_pub_uebersicht_des_vergleichs.sql'
+    ]
+}
+
+task RemoveImportedData(type: SqlExecutor, dependsOn: 'createOverview'){
```

Comment:
Diese Task evtl. mit `finalizedBy `an _calculateDifferences_ anhängen, statt mit `dependsOn` and _createOverwiev_. Hol aber noch die Meinung von Stefan ein.